h/services/search_index/_queue.py

```diff
@@ -27,6 +24,8 @@ class Result:
         DIFFERENT = "Annotations synced because they were different in Elasticsearch"
         UP_TO_DATE = "Jobs deleted because annotations were up to date in Elasticsearch"
         FORCED = "Annotations synced because their jobs had force=True"
+        TOTAL_INDEXED = "Total annotations indexed"
+        TOTAL_COMPLETED = "Total jobs completed"
```

Comment:
Of course there are only per-run totals not cumulative totals, hopefully that's clear

Reply:
I don't think we need the `/h/` part. It's not in the already existing metrics, that we already have dashboards for) and I don't think it's needed because New Relic already seems to group metrics by app. I think it does this based on the `NEW_RELIC_APP_NAME` and `NEW_RELIC_ENVIRONMENT` envvars (which also separate metrics coming from QA and prod).

Unfortunately I don't think these metrics organise into `/Indexed/` and `/Completed/` hierarchies either: when a job has `force=True` (the `FORCED` metric) we both index the annotation _and_ delete its job and we only increment the `FORCED` count.

So I've basically left these strings the same except that I've lowercased them and replaced spaces with underscores. And then I've had the Celery task prepend `"Custom/SyncAnnotations/Queue/"` to them: https://github.com/hypothesis/h/pull/6376/commits/706be0190463c09ab0bd1dff1f5bc21cb865cb9d

That actually works out quite nicely because there's nothing New Relic-y at all about the strings in `Queue`, that's all in the task.